addons/stock/static/tests/inventory_report_list.test.js

```diff
@@ -0,0 +1,203 @@
+import { defineMailModels } from "@mail/../tests/mail_test_helpers";
+import { expect, test } from "@odoo/hoot";
+import { animationFrame, queryOne } from "@odoo/hoot-dom";
+import {
+    contains,
+    defineModels,
+    fields,
+    models,
+    mountView,
+    onRpc,
+} from "@web/../tests/web_test_helpers";
+
+const { DateTime } = luxon;
+
+const arch = `
+    <list editable="top" js_class="inventory_report_list">
+        <field name="name"/>
+        <field name="age"/>
+        <field name="job"/>
+        <field name="create_date" invisible="1"/>
+        <field name="write_date" invisible="1"/>
+    </list>
+`;
+
+const setup_date = DateTime.fromISO("2022-01-03T08:03:44+00:00").toSQL();
```

Comment:
I mean, you could also convert it multiple times in other formats as well...
```suggestion
const setup_date = "2022-01-03 08:03:44"
```